packages/core/src/components/slider/multiRangeSlider.tsx

```diff
@@ -0,0 +1,239 @@
+/*
+ * Copyright 2018 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the terms of the LICENSE file distributed with this project.
+ */
+
+import classNames from "classnames";
+import * as React from "react";
+
+import { Classes, Intent } from "../../common";
+import { intentClass } from "../../common/classes";
+import * as Errors from "../../common/errors";
+import * as Utils from "../../common/utils";
+import { CoreSlider, formatPercentage, ICoreSliderProps } from "./coreSlider";
+import { Handle } from "./handle";
+import { ISliderHandleProps, SliderHandle } from "./sliderHandle";
+
+export interface IMultiRangeSliderProps extends ICoreSliderProps {
+    children?: React.ReactNode;
```

Comment:
this is not necessary as it's the same as React's definition. what would be awesome, since it's validated below, is to declare something along the lines of `children?: React.ReactElement<ISliderHandleProps>[]`